doc/release/0.4.rst

```diff
@@ -11,8 +11,12 @@ New Features
 #. ARMI now supports Python 3.12. (`PR#1813 <https://github.com/terrapower/armi/pull/1813>`_)
 #. Removing the ``tabulate`` dependency by ingesting it to ``armi.utils.tabulate``. (`PR#1811 <https://github.com/terrapower/armi/pull/1811>`_)
 #. Adding ``--skip-inspection`` flag to ``CompareCases`` CLI. (`PR#1842 <https://github.com/terrapower/armi/pull/1842>`_)
-#. Provide utilities for determining location of a rotated object in a hexagonal lattice (``getIndexOfRotatedCell``). (`PR#1846 <https://github.com/terrapower/armi/1846`)
-#. Allow merging a component with zero area into another component. (`PR#1858 <https://github.com/terrapower/armi/pull/1858>`_)
+#. Provide utilities for determining location of a rotated object in a hexagonal lattice:
+   :func:`armi.utils.hexagon.getIndexOfRotatedCell`
+   (`PR#1846 <https://github.com/terrapower/armi/1846`)
```

Comment:
```suggestion
#. Provide utilities for determining location of a rotated object in a hexagonal lattice: ``getIndexOfRotatedCell``. (`PR#1846 <https://github.com/terrapower/armi/1846`_)
```

Reply:
Minor, and unimportant cleanup:

1. This text is taken and rendered downstream without access to the API (for QA docs), so the RST links to API docs won't work.
2. It needs that `_` at the end of the URL for RST to rend the URL correctly.
3. Added a period to the end.

Reply:
Yeah you had mentioned that elsewhere and I think it got lost in resolving merge conflicts